js/main.js

```diff
@@ -46,27 +63,52 @@ function renderEntry(entry) {
   const $rest = document.createElement('div');
   $rest.className = 'column-half';
 
+  const $headerdiv = document.createElement('div');
+  $headerdiv.className = 'headerdiv';
+
   const $pic = document.createElement('img');
   $pic.src = entry.photo;
 
-  const $ename = document.createElement('p');
+  const $ename = document.createElement('span');
   $ename.textContent = entry.title;
 
   const $not = document.createElement('p');
   $not.textContent = entry.notes;
 
+  const $pen = document.createElement('i');
+  $pen.className = 'fa fa-pencil';
+
   $li.appendChild($top);
   $top.appendChild($rest);
   $top.appendChild($rested);
   $rest.appendChild($pic);
-  $rested.appendChild($ename);
+  $rested.appendChild($headerdiv);
+  $headerdiv.append($ename, $pen);
   $rested.appendChild($not);
 
   return $li;
 }
 
 const $unl = document.querySelector('ul');
 
+$unl.addEventListener('click', function (event) {
+  const $eform = document.querySelector('.eform');
+  const $newentry = document.querySelector('.new-entry');
+  if (event.target.matches('i')) {
+    for (let i = 0; i < data.entries.length; i++) {
+      // eslint-disable-next-line eqeqeq
+      if (data.entries[i].entryId == event.target.closest('li').getAttribute('data-entry-id')) {
+        data.editing = event.target.closest('li').getAttribute('data-entry-id');
```

Comment:
You should be setting the whole `data.entries[i]` object to `data.editing`. Not just the id